docs/source/user-guide/docker_files.md

```diff
@@ -6,12 +6,11 @@ Docker files are available for [simtools](https://github.com/gammasim/simtools)
 
 Types of docker files and containers available:
 
-- [simtools users](#container-for-simtools-users): a container with all software installed (CORSIKA, sim\_telarray, simtools python environment, simtools). Pull latest release with: `docker pull ghcr.io/gammasim/simtools-prod:latest`
+- [simtools users](#container-for-simtools-users): a container with all software installed (CORSIKA, sim\_telarray, simtools python environment, simtools). Pull one of the images from the [page registry (non-vector-code optimized images)](https://github.com/orgs/gammasim/packages?tab=packages&q=no_opt)
```

Comment:
I think it would be useful to also have the example like 

`docker pull ghcr.io/gammasim/simtools-prod-240205-corsika-77500-bernlohr-1.67-prod6-baseline-qgs2-no_opt:latest`

Reply:
Agree - added.